packages/next/src/build/webpack-config.ts

```diff
@@ -1985,6 +2021,10 @@ export default async function getBaseWebpackConfig(
         },
         ...(hasAppDir
           ? [
+              {
+                layer: WEBPACK_LAYERS.appRouteHandler,
+                test: /private-next-app-dir\/.*\/route\.js$/,
```

Comment:
This looks incorrect to me, shouldn't it match all possible extensions for `route`? I.e. `route.ts` too?